Apps/WebClient/src/NewClientApp/src/models/patientDataResponse.ts

```diff
@@ -5,11 +5,13 @@
 export enum PatientDataType {
     OrganDonorRegistrationStatus = "OrganDonorRegistrationStatus",
     DiagnosticImaging = "DiagnosticImaging",
+    BcCancerScreening = "BcCancerScreening",
 }
 
 export enum HealthDataType {
     OrganDonorRegistration = "OrganDonorRegistration",
     DiagnosticImagingExam = "DiagnosticImagingExam",
+    BcCancerScreeningExam = "BcCancerScreeningExam",
```

Comment:
This value mirrors the one in Patient/src/Services/IPatientDataService.cs, so if that changes, this should too.